tests/bsim/bluetooth/mesh/src/test_provision.c

```diff
@@ -1407,6 +1438,24 @@ static void test_provisioner_pb_remote_client_ncrp(void)
 		FAIL("Wrong composition data page 129");
 	}
 
+	/* Check that Composition Data Page 130 still exists and is now equal to Page 2. */
```

Comment:
These checks are repeated for each page pair. Could this be wrapped into a function?

Reply:
Yes, I will add it in the other PR 👍